dev-libs/aws-c-event-stream/aws-c-event-stream-0.2.7.ebuild

```diff
@@ -0,0 +1,45 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake
+
+DESCRIPTION="C99 implementation of the vnd.amazon.eventstream content-type"
+HOMEPAGE="https://github.com/awslabs/aws-c-event-stream"
+SRC_URI="https://github.com/awslabs/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="static-libs test"
+
+RESTRICT="!test? ( test )"
+
+DEPEND="
+	>=dev-libs/aws-c-common-0.5.2:=[static-libs=]
+	>=dev-libs/aws-c-io-0.9.2:=[static-libs=]
+	>=dev-libs/aws-checksums-0.1.10:=[static-libs=]
```

Comment:
I think `[static-libs=]` is unnecessary and you'll want to use `?` instead.

Reply:
I have taken this from some other ebuilds in the tree. How would these lines be better written? I have to admit I do not know what you mean by "use ? instead" exactly.

Reply:
`[static-libs?]` will require `+static-libs` on a dep if it's enabled for this ebuild, while 
`[static-libs=]` will need the dep match to match use flag for this ebuild. So to say, In order for me to test, or anyone to install this package with `+static-libs`, it needs to be enabled on all the specified deps (correct), but if I want to test `-static-libs` or install that afterward, all the deps also need to be recompiled with `-static-libs`. I doubt that is **required**.

Reply:
Ooohhh! Alright, I will fix that with the next bump. Which will be this month already as AWS has a really quick release cycle.
Thank you very much for all your help and support!